cordapp/src/test/resources/net/corda/plugins/CorDappWithContractInfo.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id 'java'
+    id 'net.corda.plugins.cordapp'
+}
+
+repositories {
```

Comment:
Perhaps refactor some of the repeated elements from these test scripts into common files? E.g. you could create a `repositories.gradle` file in `test/resources` containing:
```
repositories {
    mavenCentral()
    maven { url 'https://blah.blah.blah' }
}
```
Then replaces each `repositories` block with
```
apply from: 'repositories.gradle`
```
Finally, you'd need just need to copy this `repositories.gradle` file into the `testProjectDir`. You could do this in the `CordappTest`'s `@Before` function.

The point is that we can then update these repositories in just one place rather than `N`.

Reply:
Done!